src/main/java/org/sagebionetworks/bridge/workerPlatform/config/SpringConfig.java

```diff
@@ -66,6 +67,7 @@
         "org.sagebionetworks.bridge.udd",
         "org.sagebionetworks.bridge.uploadredrive",
         "org.sagebionetworks.bridge.workerPlatform",
+        "org.sagebionetworks.bridge.participantRoster" //TODO not sure I need this one or the bean below
```

Comment:
This is correct, because we want to do ComponentScan on the whole package. The SpringConfig files listed below are for older legacy workers.